src/maml/algorithm/ppo.py

```diff
@@ -2,11 +2,11 @@
 Proximal Policy Optimization algorithm implementation for training
 """
 
+import copy
```

Comment:
```suggestion
from copy import deepcopy
```
로 쓰고, 바로 deepcopy를 이용하는게 더 명확한 것 같아요.

Reply:
아래 5dd585c commit에서 `self.old_policy` 가 삭제됨에 따라 `deepcopy`도 삭제되었습니다